localstack/utils/bootstrap.py

```diff
@@ -282,16 +274,7 @@ def get_preloaded_services() -> Set[str]:
 
 
 def should_eager_load_api(api: str) -> bool:
-    apis = get_preloaded_services()
-
-    if api in apis:
-        return True
-
-    for enabled_api in apis:
-        if api.startswith(f"{enabled_api}:"):
-            return True
-
-    return False
+    return api in get_preloaded_services()
```

Comment:
Unfortunately, I haven't seen the remaining part of the function when I did the initial PR review (suggesting the simplification of this function). I think it might be a bit hasty, based on my mistaken comment, to break this behavior. I have just recently seen a `SERVICES` config which still contains port mappings, and I think there's no harm in continuing to just ignore them.
Sorry for the inconvenience and the confusion with the initial PR comment...